tests/UI/campaigns/utils/files.js

```diff
@@ -150,4 +150,16 @@ module.exports = {
       fs.writeFileSync(imageName, image.data);
     });
   },
+
+  /**
+   * Rename files
+   * @param oldPath
+   * @param newPath
+   * @return {Promise<void>}
+   */
+  async renameFile(oldPath, newPath) {
```

Comment:
We didn't have this method before ?

Reply:
@boubkerbribri Does this method already exist ?

Reply:
No it's a new one